kit/KitQueue.hpp

```diff
@@ -42,11 +42,53 @@ class KitQueue
         put(Payload(value.data(), value.data() + value.size()));
     }
 
+    struct Callback {
+        int _view; // -1 for all
+        int _type;
+        std::string _payload;
+
+        Callback() : _view(-1), _type(-1) { }
+        Callback(int view, int type, const std::string payload) :
+            _view(view), _type(type), _payload(payload) { }
+
+        static std::string toString(int view, int type, const std::string payload)
+        {
+            std::ostringstream str;
+            str << view << " " << lokCallbackTypeToString(type) << " "
```

Comment:
iOS build seems to suggest that #include <LibreOfficeKit/LibreOfficeKitEnums.h> is newly needed to build